cookbooks/aws-parallelcluster-platform/resources/fabric_manager/partial/_fabric_manager_install_debian.rb

```diff
@@ -35,9 +35,14 @@
 end
 
 def arch_suffix
+  arm_instance? ? 'sbsa' : 'x86_64'
+end
+
+def name_suffix
   arm_instance? ? 'arm64' : 'amd64'
 end
 
 def fabric_manager_url
-  "#{node['cluster']['artifacts_s3_url']}/dependencies/nvidia_fabric/#{platform}/#{fabric_manager_package}_#{fabric_manager_version}-1_#{arch_suffix}.deb"
+  # "#{node['cluster']['artifacts_s3_url']}/dependencies/nvidia_fabric/#{platform}/#{fabric_manager_package}_#{fabric_manager_version}-1_#{arch_suffix}.deb"
+  "https://developer.download.nvidia.com/compute/cuda/repos/#{platform}/#{arch_suffix}/#{fabric_manager_package}_#{fabric_manager_version}-1_#{name_suffix}.deb"
```

Comment:
Sync we are going to remove this hardocde value after testing

Reply:
Yes this is same as https://github.com/aws/aws-parallelcluster-cookbook/pull/2996#discussion_r2240969910